setup.py

```diff
@@ -4,7 +4,7 @@
     "fakeredis~=0.11.0",
     "cachetools~=4.1.0",
     "python-consul~=1.0.1",
-    "redis~=2.10.6",
+    "redis>=2.10.6,<3.6",
```

Comment:
What potential incompatibilities are we looking at? I'm wondering if this necessitates a major version bump for the package.

Reply:
Going through the release notes / code now.

Reply:
Specifically https://github.com/andymccurdy/redis-py#upgrading-from-redis-py-2x-to-30

Reply:
I don't see any breaking changes in the changelog. We do use `StrictRedis` as a type annotation but it looks like there's an alias. It also wouldn't be hard to switch to `Redis`.

Reply:
Why not `<4.0`? Should be able to trust `redis` to not introduce breaking changes in minor releases.